src/automationttsengine/AutomationTtsEngine.vcxproj

```diff
@@ -167,10 +167,6 @@
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <TargetMachine>MachineX86</TargetMachine>
     </Link>
-    <PostBuildEvent>
-      <Message>Performing Registration</Message>
-      <Command>regsvr32 /s /c "$(TargetPath)"</Command>
-    </PostBuildEvent>
```

Comment:
@mzgoddard I'm still only guessing about why gh-9 removes this registration, but [I hesitated to accept the removal in that context](https://github.com/bocoup/at-automation-experiment/pull/9/files#r769840421) because it's important for the C++ development workflow.

Since this patch relocates the registration to MakeVoice.exe, and since that doesn't be appear to be used in gh-9, I think this will address both of our needs. And (as noted in the commit message) it gets us closer to a solution where `npm install` just shells out to `MakeVoice.exe`. Although I wonder if that will have a negative impact on what you're trying to achieve in gh-9.

Reply:
I recall from the difficulty getting the dll working on an image I created that the logging functionality I added to the dll locally that it'd initialize the dll scoped logging class instance when the dll is loaded. Not that logging functionality but some other change made by regsvr32 called while building ttseng encountered an error in that context.

A result from a [quick google search](https://www.google.com/search?client=firefox-b-1-d&q=regsvr32+exit+code+3) seems to confirm that:
> If something goes wrong, the regsvr32 program reports the error details to the user, and the exit code summarizes which step it got stuck at.<br>...<br>What do the various regsvr32 exit codes mean?
>
> Exit code | Meaning
> -- | --
> 3 | LoadLibrary failed